packages/react-components/src/components/AppFrame/AppFrame.stories.tsx

```diff
@@ -0,0 +1,223 @@
+              }}
+              isActive={activeItem === 'tickets'}
+              href="#"
+            />
+            <NavBarItem
+              label="Team"
+              icon={<Icon source={Icons.People} />}
+              onClick={(e) => {
+                e.preventDefault();
+                setActiveItem('team');
+              }}
+              isActive={activeItem === 'team'}
+              href="#"
+            />
+            <NavBarItem
+              label="Reports"
+              icon={<Icon source={Icons.Report} />}
+              onClick={(e) => {
+                e.preventDefault();
+                setActiveItem('reports');
+              }}
+              isActive={activeItem === 'reports'}
+              href="#"
+            />
+            <NavBarItem
+              label="Apps"
+              icon={<Icon source={Icons.Apps} />}
+              onClick={(e) => {
+                e.preventDefault();
+                setActiveItem('apps');
+              }}
+              isActive={activeItem === 'apps'}
+              href="#"
+            />
+          </NavBarList>
+        }
+        bottomNavBarNode={
+          <NavBarList>
+            <NavBarItem
+              label="Billing"
+              icon={<Icon source={Icons.CreditCardOutline} />}
+              onClick={(e) => {
+                e.preventDefault();
+                setActiveItem('billing');
+              }}
+              isActive={activeItem === 'billing'}
+              href="#"
+            />
+            <NavBarItem
+              label="Settings"
+              icon={<Icon source={Icons.Settings} />}
+              onClick={(e) => {
+                e.preventDefault();
+                setActiveItem('settings');
+              }}
+              isActive={activeItem === 'settings'}
+              href="#"
+            />
+            <NavBarItem
+              label="News"
+              icon={<Icon source={Icons.Notifications} />}
+              onClick={(e) => {
+                e.preventDefault();
+                setActiveItem('news');
+              }}
+              isActive={activeItem === 'news'}
+              href="#"
+            />
+            <NavBarItem
+              label="User"
+              disableOpacity
+              disableTooltip
+              icon={
+                <Tooltip
+                  floatingStrategy="fixed"
+                  placement="right"
+                  triggerRenderer={
+                    <Avatar
+                      status="available"
+                      type="image"
+                      src={defaultImage}
+                      alt="User avatar"
+                    />
+                  }
+                >
+                  Custom element with tooltip
+                </Tooltip>
+              }
+              onClick={(e) => e.preventDefault()}
+              href="#"
+            />
+          </NavBarList>
+        }
+        subNavBar={getSubNav()}
+        topBarNode={
```

Comment:
we can just name it - `topBar`, we do not need to add a "Node" suffix, it would be clear from TS typing declaration.